mod/friendica.php

```diff
@@ -42,7 +42,7 @@ function friendica_init(App $a)
 
 		Config::load('feature_lock');
 		$locked_features = [];
-		if (is_array($a->config['feature_lock']) && count($a->config['feature_lock'])) {
+		if (!empty($a->config['feature_lock']) && count($a->config['feature_lock'])) {
```

Comment:
Can you easily extract the code quality commits from this PR?

Reply:
I don't know how. I cannot change this branch due to the database changes.

Reply:
Ok, no problem.